lib/ansible/module_utils/crypto.py

```diff
@@ -46,3 +48,68 @@ def get_fingerprint(path):
         pass
 
     return fingerprint
+
+
+def load_privatekey(path, passphrase=None):
+    """Load the specified OpenSSL private key."""
+
+    try:
+        privatekey_content = open(path, 'r').read()
+        privatekey = crypto.load_privatekey(crypto.FILETYPE_PEM,
+                                            privatekey_content,
+                                            passphrase)
+        return privatekey
+    except (IOError, OSError) as exc:
+        raise OpenSSLModuleError(exc)
+
+
+def load_certificate(path):
+    """Load the specified certificate."""
+
+    try:
+        cert_content = open(path, 'r').read()
+        cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert_content)
+        return cert
+    except (IOError, OSError) as exc:
+        raise OpenSSLModuleError(exc)
+
+
+class OpenSSLModuleError(Exception):
+    pass
```

Comment:
exception classes should be defined after impotrs and constants and before helper functions.